src/PostgREST/Types.hs

```diff
@@ -182,7 +182,8 @@ type Field = (FieldName, Maybe JsonPath)
 type Alias = Text
 type Cast = Text
 type NodeName = Text
-type SelectItem = (Field, Maybe Cast, Maybe Alias)
+type RelationDetail = Text
```

Comment:
This type is not clear to me, Relation is already a confusing name since PostgreSQL calls tables and views "relations". Could this name be more precise?